src/main/java/com/cloudbees/jenkins/GitHubTrigger.java

```diff
@@ -15,6 +15,7 @@
  * and triggers a build.
  *
  * @author aaronwalker
+ * @deprecated extend {@link GitHubTrigger2} instead
```

Comment:
Careful—does this produce deprecation warnings for implementers of `GitHubTrigger2`?